### DISCUSSION TITLE
Close : Translation for future release.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#143] [General]
[2021-12-13_Translation_biblioteq_fr-FR.tar.gz](https://github.com/textbrowser/biblioteq/files/7704954/2021-12-13_Translation_biblioteq_fr-FR.tar.gz)

**Comment by textbrowser** [comment]
:) Will add it as soon as I can.

**Reply by textbrowser** [reply]
Added!

**Comment by meteos77** [comment]
Translation of the user guide

Context: I participate to another opensource project : Universal Blood Pressure Manager "https://codeberg.org/LazyT/ubpm", advertising :-) as a user/translator and yesterday I made the translation of the user guide.

I find the way of translating more pleasant than browsing with LibreOffice the pages to see where you made changes between versions.

On the project, the translations are done as for the other menu translations. 

suggestion: to be able to have the fields of the documentation to translate with linguist it will be great.

proposal : I do not know how to do for the installation of all the internal system but after I can ensure the resumption for the passage of the files .odt English version towards new system qhc qch qhcp and qhp as well as its translation in French.

**Comment by meteos77** [comment]
today's updat
[2022-01-17_Translation_biblioteq_fr_FR.ts.zip](https://github.com/textbrowser/biblioteq/files/7883353/2022-01-17_Translation_biblioteq_fr_FR.ts.zip)
e

**Comment by textbrowser** [comment]
Installed.

**Comment by meteos77** [comment]
French language package for the March 30 version. Thanks for the integration
[French-Package-edition-March 30, 2022.tar.gz](https://github.com/textbrowser/biblioteq/files/8357898/French-Package-edition-March.30.2022.tar.gz)

**Comment by textbrowser** [comment]
Committed!

**Comment by meteos77** [comment]
Documentation in xhtml version 1.19 optimized : size 1.2MB
[BiblioteQ-Fr.html.tar.gz](https://github.com/textbrowser/biblioteq/files/8441910/BiblioteQ-Fr.html.tar.gz)

currently the link concerning the french documentation leads to the presentation page of the french biblioteQ project
